docs/knowledgebase/advanced/cryptography.md

```diff
@@ -35,11 +35,11 @@ Substrate uses the [`blake2`](https://docs.rs/blake2/) implementation in Rust.
 
 ## Public-Key Cryptography
 
-Public-key cryptography is used in Substrate to provide a robust authentication system.
+Public-key cryptography is used in Substrate to provide a robust authentication system. The application of a public key is commonly utilized in encrypting communication to provide confidentiality by obfuscating the recipient's public key which can only be decrypted by the recipient's paired private key.
```

Comment:
Revert, has nothing to do with encryption.

Reply:
correct that should of read private.. obviously I will never edit anything prior to coffee again